main/medium/src/misc_newtypes.rs

```diff
@@ -1118,6 +1118,65 @@ impl TryFrom<f64> for PositionInQuarterNotes {
     }
 }
 
+/// This represents a position expressed as an amount of midi ticks (PPQ).
```

Comment:
Hope I'm not too pedantic ;)

```suggestion
/// This represents a position expressed as an amount of MIDI ticks (PPQ).
```